runtime/src/main/kotlin/kotlin/collections/ArrayUtil.kt

```diff
@@ -75,10 +76,42 @@ internal fun <E> Array<E>.resetAt(index: Int) {
 
 @SymbolName("Kotlin_Array_fillImpl")
 @PointsTo(0b01000, 0, 0, 0b00001) // <array> points to <value>, <value> points to <array>.
```

Comment:
What should I specify here, if I want to change parameter order to `array, value, from, to`?

Reply:
Just remove PointsTo, escape analysis need to be reworked.

Reply:
Ok, good to know. I've decided not to change it though.